main.py

```diff
@@ -23,5 +30,7 @@
             exc = '{}: {}'.format(type(e).__name__, e)
             print('Failed to load extension {}\n{}'.format(ext, exc))
 
-        bot.run('')
+        # To run: Make a file called key.text in root
+        file = open("key.txt", "r")
+        bot.run(file.read())
 
```

Comment:
1. Enklere for folk å teste sette opp selv. 
2. For å slippe å fjerne key ved commits. 

Det var egentlig den første ideen jeg kom på for å fjerne nøkkelen fra commits, finnes det en bedre løsning?